### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by tobiaslieber**
(probably not exhaustive)

### CODE REVIEW

File: docs/api.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
         <a style ="color:orangered" href="#events"> Events</a> &nbsp; |  &nbsp;
         <a style ="color:orangered" href="#slots"> Slots</a> &nbsp; |  &nbsp;
         <a style ="color:orangered" href="./css-parts.html"> Advance Styling </a> &nbsp; |  &nbsp;
```
Comment (waldyrious) [inline docs/api.html:76]: You might want to fix this one as well :)

```suggestion
        <a style ="color:orangered" href="./css-parts.html"> Advanced Styling </a> &nbsp; |  &nbsp;
```

Comment (waldyrious) [reply docs/api.html:76]: The same typo is present here as well, btw:

https://github.com/mrin9/RapiDoc/blob/2ab68d23eb66b7d04605498302546f38cdb0be57/docs/list.html#L185

**Comment by mrin9** [comment]
thanks for taking time to provide fixes and the reviews !!!

**Comment by waldyrious** [comment]
Nice expansion of the PR, @tobiaslieber! Great eye :)

**Pull request merged**